tgui/packages/tgui/interfaces/Photocopier.js

```diff
@@ -97,15 +97,26 @@ const Actions = (props, context) => {
         onClick={() => act('scandocument')}
       />
       {!!issilicon && (
-        <Button
-          fluid
-          icon="image"
-          color="green"
-          float="center"
-          textAlign="center"
-          content="Print from database"
-          onClick={() => act('ai_pic')}
-        />
+        <div>
```

Comment:
I don't believe you need this `<div>` , as the buttons are already contained within a fragment and the `<div>` is not being used to apply a selector.

Reply:
yeah, no
it is a requirement
`Parse Error: Adjacent JSX elements must be wrapped in an enclosing tag`

Reply:
This could be wrapped in a fragment `<></>` instead of a `<div>` most likely.  `<div>` works here, but a fragment would most likely be better.